packages/checkPrescriptionStatusUpdates/src/checkPrescriptionStatusUpdates.ts

```diff
@@ -58,31 +58,43 @@ const lambdaHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPro
     exclusiveStartKeyPrescriptionID: event.headers["exclusivestartkey-prescriptionid"],
     exclusiveStartKeyTaskID: event.headers["exclusivestartkey-taskid"]
   }
-  const queryResult = await getItemStatusUpdates(inputData, logger)
 
-  let statusCode = 200
   const result = {
     items: []
   }
-  if (queryResult.Count === 0) {
-    statusCode = 404
-  } else {
-    result.items = queryResult.Items
-  }
-
   const headers = {
     "Content-Type": "application/json",
     "Cache-Control": "no-cache"
   }
 
-  if (queryResult.LastEvaluatedKey) {
-    for (const key in queryResult.LastEvaluatedKey) {
-      headers[`LastEvaluatedKey-${key}`] = queryResult.LastEvaluatedKey[key]
+  const MIN_RESULTS_RETURNED = 5
+  const MAX_RESULTS_RETURNED = 15
+  for (;;) {
```

Comment:
better for this to be a while loop

Reply:
I thought the same, apparently that's the semantic way of doing it though 🤔 

https://eslint.org/docs/latest/rules/no-constant-condition

Reply:
Changed my mind, while(true) is worth the eslintrc change
e9850b2a0b3ceb15fefd2742cebe7ab9f1bca681